sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.2.0-beta.2 (Unreleased)
-- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies
+- Reverted change in 1.2.0-beta.1 which moved @rollup/plugin-json from devDependencies to dependencies.
```

Comment:
Can we include the information on why the revert was needed?

Reply:
@jonathandturner I am not aware of why we had to revert this. Help is appreciated.

Reply:
This would have been a breaking change to roll out as a dependency as users would have needed to modify their projects to upgrade. Since this functionality wasn't strictly required, we moved it back to being a devDependency

Reply:
Did we move it to devDependencies as part of 1.2.0-beta.1? I thought we never released that.

Reply:
`"@rollup/plugin-json": "^4.0.0"` Is definitely part of `1.2.0-beta.1`, as I just checked. Can I remove this changelog entry?

Reply:
welp, my brain was reading that backwards. This changelog entry means that 1.2.0-beta.1 had `@rollup/plugin-json`, and 1.2.0-beta.2 won't have it.

Another way to phrase it could be:

> Removed `@rollup/plugin-json` from being a required dependency.

Thoughts? I'm not feeling very creative today.

Reply:
Ok after re-reading the request and getting some feedback from Jonathan, I'll push this change:

> Reverted a change in 1.2.0-beta.1 which moved `@rollup/plugin-json` from `devDependencies` to `dependencies`. `@rollup/plugin-json` was placed as a dependency due to an oversight, and it is not necessary for `@azure/identity`.

Reply:
Resolving for now.